R/utils.R

```diff
@@ -0,0 +1,23 @@
+#' Read in a epidist Stan code chunk
+#'
+#' @param path The path within the "stan" folder of the installed epidist
+#' package to the Stan code chunk of interest.
+#' @return A character string containing the Stan code chunk of interest.
+#' @family utils
+#' @export
+epidist_stan_chunk <- function(path) {
+  local_path <- system.file(paste0("stan/", path), package = "epidist")
+  paste(readLines(local_path), collapse = "\n")
+}
+
+#' Label a epidist Stan model with a version indicator
+#'
+#' @return A brms stanvar chunk containing the package version used to build
+#' the Stan code.
+#' @family utils
+#' @export
+epidist_version_stanvar <- function() {
+  version <- utils::packageVersion("epidist")
+  comment <- paste0("// code chunks used from epidist ", version, "\n")
+  stanvar(scode = comment, block = "functions")
```

Comment:
not using `brms::` or in namespace - likely an issue

Reply:
Added `brms::`